packages/chain-adapters/src/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -55,11 +61,29 @@ jest.mock('axios', () => ({
 describe('EthereumChainAdapter', () => {
   let args: ethereum.ChainAdapterArgs = {} as any
 
+  const gasPrice = '42'
+  const gasLimit = '42000'
+  const erc20ContractAddress = '0xc770eefad204b5180df6a14ee197d99d808ee52d'
+  const value = 400
+
+  const chainSpecificWithoutErc20ContractAddress = {
+    gasPrice,
+    gasLimit
+  }
+
+  const chainSpecificWithErc20ContractAddress = {
+    ...chainSpecificWithoutErc20ContractAddress,
+    erc20ContractAddress
+  }
+  const getInfoMockResponse = {
+    data: { network: 'mainnet' }
+  }
```

Comment:
In order to avoid accidental mutations of these objects, these should be functions that return a copy of the object.

I pattern that I like to follow is:
```ts
const mockWhateverData = (obj: CorrectType) => _.merge({
  prop: 1,
  nested: {
    prop: 2
  }
}, obj)
```

this allows each test to call `mockWhateverData()` to get a clean copy of the mock or `mockWhateverData({ prop: false })` to change a specific property of the default mock to test something slightly differently.